Project.toml

```diff
@@ -0,0 +1,17 @@
+name = "KeyedFrames"
+uuid = "ddd84059-9b64-5d1b-85eb-da48df743e32"
+version = "1.0.0"
+
+[deps]
+DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
+
+[compat]
+DataFrames = ">= 0.11.6"
```

Comment:
Because using `delete!` not `deletecols!` for deleting a column,
that implies the version must be above I thin 0.17 or maybne 0.16,
can you check the releases of DataFrames.jl and find that out, then amend this?